marimbabot_vision/scripts/data_augmentation.py

```diff
@@ -0,0 +1,50 @@
+from generate_data import NUM_SAMPLES, NUM_WORKER, OUTPUT_DIR
+from multiprocessing import Pool
+from PIL import Image
+
+import os
+import tqdm
+import shutil
+import albumentations as A
```

Comment:
Not a mayor issue, but can you sort the imports? In VSCode it is just right click -> sort imports

Reply:
that does not work for me